web_src/css/modules/animations.css

```diff
@@ -18,7 +18,6 @@
   position: absolute;
   display: block;
   height: 4rem;
-  max-height: 50%;
```

Comment:
Without max-height, does it work with when container's height < 4em?

Reply:
Works now with `100%` in both directions it seems.